components/menu/menu-item-link.js

```diff
@@ -61,6 +61,7 @@ class MenuItemLink extends MenuItemMixin(LitElement) {
 	}
 
 	_onClick(e) {
+		this.shadowRoot.querySelector('a').dispatchEvent(new CustomEvent('click', { bubbles: false }));
```

Comment:
Should this happen even if they want to prevent default?  Since that will cancel the click event on the menu item, but not this new event we are dispatching?

Reply:
Yeah, similar to the keydown scenario below, which comes into play when the menu-item gets the focus and the user hits enter/space. It's code makes me suspicious that there could be an issue with ghost-clicking.